tests/gdtest/gdtest.c

```diff
@@ -203,7 +212,7 @@ mkdtemp (char *tmpl)
 	static int counter = 0;
 	char *XXXXXX;
 	struct timeval tv;
-	_int64 value;
+	__int64 value;
```

Comment:
Similar to `Ui64`, `_int64` is only avaible under MSVC, while `__int64` is avaible under both MSVC and MinGW.